public/js/components/SourceFooter.js

```diff
@@ -96,7 +96,7 @@ const SourceFooter = React.createClass({
         (!isEnabled("prettyPrint")
           && !isEnabled("blackBox")
           && !isEnabled("search"))) {
-      return null;
+      return dom.div({ className: "source-footer" });
```

Comment:
This is probably fine, given the next release is likely to include source search. 

I could add a JS condition to hide the footer and increase the editor height, but that doesn't seem worth it.